src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/basic/PortalSpawnModifier.java

```diff
@@ -55,11 +58,41 @@ public void portalSpawn(CreatureSpawnEvent event) {
 				event.setCancelled(true);
 				return;
 			}
-			if (roll(ghastChance)) {
+
+			//Disabled since we don't want ghasts spawning in nether portals.
+			//if (roll(ghastChance)) {
+			//	event.setCancelled(true);
+			//	spawnGhast(event.getLocation());
+			//}
+		}
+	}
+
+	//Wither Skeleton + Ghast spawns
+	@EventHandler
+	private void hellSpawn(CreatureSpawnEvent event) {
+		World world = event.getLocation().getWorld();
+		if (event.getSpawnReason() == SpawnReason.NETHER_PORTAL) {
```

Comment:
This will make this code apply to all kinds of custom spawning (plugin based, spawn eggs etc.), definitely not what we want.

Reply:
that is why the following if statements ensure that it isn't modifying other mob behavior, this is exactly what Humbug's old code contained.